src/main/java/org/molgenis/vcf/decisiontree/runner/DecisionTreeFactoryImpl.java

```diff
@@ -213,12 +213,15 @@ private BoolMultiQuery toBoolClause(VcfMetadata vcfMetadata,
         .map(query -> toBoolQuery(vcfMetadata, query, files)).collect(
             Collectors.toList());
     return BoolMultiQuery.builder().id(configBoolMultiQuery.getId()).queryList(queries)
-        .operator(toClauseOperator(configBoolMultiQuery.getOperator()))
+        .operator(toMultiQueryOperator(configBoolMultiQuery.getOperator()))
         .build();
   }
 
-  private BoolMultiQuery.Operator toClauseOperator(ConfigClauseOperator configOperator) {
+  private BoolMultiQuery.Operator toMultiQueryOperator(ConfigClauseOperator configOperator) {
     BoolMultiQuery.Operator operator;
+    if (configOperator == null) {
```

Comment:
What is the expected behavior when using a BoolMultiQuery with multiple queries but no operator?

Reply:
An exception will be thrown:

```
        if (outcome.getOperator() == null) {
          throw new ConfigDecisionTreeValidationException(String.format(
              "MultiBool node '%s' contains an outcome with multiple queries but without an operator.",
              id));
        }
```